unpackaged/config/trial/workflows/Contact.workflow

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workflow xmlns="http://soap.sforce.com/2006/04/metadata">
+    <rules>
+        <fullName>npe01__Contact%2EEmailChanged_Alternate</fullName>
```

Comment:
Aren't these already packaged in the NPE01 child package? I didn't think we could deploy changes to the metadata if they're included in another managed package. Basically, these aren't subscriber editable so I think they're only creatable/updatable through the NPE01 package install/update.

Reply:
These are here to activate the workflows because they don't get activated when installed as a managed package. It works because we aren't changing anything other than activation status compared to the npe01-packaged metadata. It would be nice if the MD-API would let us only include the change to the activation state here, but it complains if I leave other parts out.